lib/scnnr/recognition.rb

```diff
@@ -24,7 +25,9 @@ def error?
     end
 
     def to_h
-      { 'id' => self.id, 'objects' => self.objects.map(&:to_h), 'state' => self.state.to_s }
+      hash = { 'id' => self.id, 'objects' => self.objects.map(&:to_h), 'state' => self.state.to_s }
```

Comment:
I think `hash = { 'id' => self.id, 'objects' => self.objects.map(&:to_h), 'state' => self.state.to_s, 'image' => self.image&.to_h }` is shorter and better. What do you think?

Reply:
Sorry I means as follows:

```
def to_h
  { 'id' => self.id, 'objects' => self.objects.map(&:to_h), 'state' => self.state.to_s, 'image' => self.image&.to_h }.compact
end
```